docs/source/pillars/information_governance.md

```diff
@@ -91,6 +91,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
   - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
   - Recommended
+* - You log data access queries and publish these logs
+  - Transparency is a key principle for an environment that is trusted for research.
+    With these logs publicly available, there is transparency around how the data was used, and any actions that could compromise the security of the sensitive data.
+  - Recommended
+* - You log user actions other than data access, for example network connection attempts or infrastructure actions within a TRE.
```

Comment:
Not sure how to comply with this. If you log _anything_ other than data access to you comply? Do you need to log _everything_ other than data access?

Reply:
Agree this should be more tangible, e.g. laying out all the other possible things we think you should be able to log if being exceptional